### ISSUE TITLE
[beta4] wrong URL in UI for generic webhook

### ISSUE THREAD

**Issue by thoraxe** [openshift/origin#2981]
https://github.com/openshift/training/issues/257

```
https://ose3-master.example.com:8443/osapi/v1beta1/buildConfigHooks/ruby-sample-build//github?namespace=wiring 
```

This results in a 301/404:

```
[alice@ose3-master beta4]$     curl -i -L -H "Accept: application/json"     -H "X-HTTP-Method-Override: PUT" -X POST -k     https://ose3-master.example.com:8443/osapi/v1beta1/buildConfigHooks/ruby-sample-build//
github?namespace=wiring
HTTP/1.1 301 Moved Permanently
Location: /osapi/v1beta1/buildConfigHooks/ruby-sample-build/github?namespace=wiring
Date: Tue, 09 Jun 2015 01:20:12 GMT
Content-Length: 0
Content-Type: text/plain; charset=utf-8

HTTP/1.1 404 Not Found
Content-Type: text/plain; charset=utf-8
Date: Tue, 09 Jun 2015 01:20:12 GMT
Content-Length: 40

unexpected URL ruby-sample-build/github
```

```
[alice@ose3-master beta4]$ osc version
osc v0.5.2.2-26-g701be15
kubernetes v0.17.1-804-g496be63
```

**Comment by liggitt** [comment]
Fixed in #2983 

Correct format is
```
https://localhost:8443/osapi/v1beta3/namespaces/wiring/buildconfigs/ruby-sample-build/webhooks/secret101/generic
```

**Comment by liggitt** [comment]
The 301 is a red herring trying to normalize the `//` to a single `/`
